repos/system_upgrade/el7toel8/actors/kernel/checkkerneldrivers/files/removed_drivers.txt

```diff
@@ -116,88 +78,22 @@ pata_serverworks
 pata_sil680
 pata_sis
 pata_via
-pcc-cpufreq
-pch_dma
-pch_gbe
-pch_phub
-pci-hyperv
-pcnet32
 pdc_adma
-pm80xx
+pm80xx(pm8001)
```

Comment:
@examon  This seems incorrect.
```suggestion
pm80xx
```

With previous string, you would not be able to find pm80xx.